components/backgroundlayers/index.css

```diff
@@ -0,0 +1,63 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* used in editing and browsing contexts */
+.spectrum-BackgroundLayers--elevated {
+	--spectum-backgroundlayers-background-color: var(--spectrum-gray-25);
+	--spectrum-backgroundlayers-shadow-horizontal: 0;
+	--spectrum-backgroundlayers-shadow-vertical: 0;
+	--spectrum-backgroundlayers-shadow-blur: 5px;
+	--spectrum-backgroundlayers-shadow-color: var(--spectrum-gray-600);
+	.spectrum--dark & {
+		--spectum-backgroundlayers-background-color: var(--spectrum-gray-75);
+		--spectrum-backgroundlayers-shadow-color: var(--spectrum-gray-100);
+	}
+
+	background-color: var(--spectum-backgroundlayers-background-color);
+	filter: drop-shadow(
```

Comment:
Is there a benefit to using `filter: drop-shadow` instead of `box-shadow`? I am thinking about if the class is ever applied to an image, the shadow will follow the contours of the alpha channel, which may or may not be desirable.

Reply:
thanks for suggesting this, I've changed it to a box-shadow